lib/api/controllers/authController.js

```diff
@@ -81,8 +84,23 @@ class AuthController {
     assertArgsHasAttribute(request, 'strategy');
 
     const strategy = request.input.args.strategy;
+    const passportRequest = new IncomingMessage();
+
+    passportRequest.url = `/login?strategy=${strategy}`;
+    passportRequest.method = 'POST';
```

Comment:
I'll check if it can be retrieved from the request for http and keep it as POST (and add some comment about it).
The same applies for the url. The idea was just to create an as-valid-as-possible object here.